### DISCUSSION TITLE
How to change simple mode to create database.

### DISCUSSION THREAD

**Discussion by gilbertttsubay** [dbeaver/dbeaver#13514] [Q&A]
Can you help me to solve "Object create not available in simple view mode"
![image](https://user-images.githubusercontent.com/35297439/129209953-e688629f-0047-4b5a-b988-ec8903f45977.png)

there are some guys who solved it with "Hello I also had same problem.Just right click-edit connection-navigator view and change to advanced."

but i dont see the "navigatior view".
![image](https://user-images.githubusercontent.com/35297439/129210184-afd071c4-f9fd-4527-9ef2-09bccd092344.png)
and where to change to advanced?

please just show or screenshot the image

**Comment by LonwoLonwo** [comment]
Hello @gilbertttsubay 

You can change it in Connection Settings -> General -> Navigator view

![2021-08-12 17_07_24-DBeaver Enterprise 21 1 5 - mood_data](https://user-images.githubusercontent.com/45152336/129211637-a6b95bbd-5227-414d-a6bf-6f0623abc803.png)

Or use right-click on a connection in Database Navigator and choose "Connection view".

![2021-08-12 17_06_38-DBeaver Enterprise 21 1 5 - mood_data](https://user-images.githubusercontent.com/45152336/129211792-a4f34943-9abe-4c28-8fe6-de4b76978ac2.png)

**Comment by gilbertttsubay** [comment]
Yeah i have done it @LonwoLonwo thank you very much
